### PR TITLE
Update the existing "Settings" screen to match the new design

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/0/1204667877992124/f

### Description
Updated the Settings screen.

### Steps to test this PR

_Settings - Privacy section_

- [ ] Install from this branch and go to the `Settings` screen.
- [ ] Notice the items are the same as in Design (the sections / order / copy): Default Browser, Private Search, Web Tracking Protection, Cookie Pop-Up Protection, AppTP, NetP.
- [ ] Tap on `Default Browser` and notice the screen to choose the default browser is opened (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Private Search` and notice the new Private Search screen looks as in design (icon and copy at the top, Search Settings title, Autocomplete Suggestions, More Search Settings). 
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Web Tracking Protection` and notice the new Web Tracking Protection screen looks as in design (icon and copy at the top, GPC and Unprotected Sites).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Cookie Pop-Up Protection` and notice the new Cookie Pop-Up Protection screen looks as in design (copy at the top, Let DDG handle cookie pop-ups toggle). 
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Email Protection` and notice you are sent to the browser tab (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `App Tracking Protection` and notice AppTP onboarding is opened (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Network Protection` and notice the waitlist screen is opened (as in Internal).
- [ ] Go back to the Settings screen. 

_Settings - Settings section_

- [ ] Tap on `Autofill Logins` and notice Logins is opened (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Sync` and notice Sync is opened (as in Internal).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Home Screen Widget` and notice the add widget prompt (as in Production).
- [ ] Tap on `Fire Button` and notice the new Fire Button screen looks as in design (Fireproof Sites, Automatically Clear, Clear On, Fire Button Animation).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Permissions` and notice the new Permissions screen looks as in design (Site Permissions, Notifications, Open Links in Apps).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Appearance` and notice the new Appearance looks as in design (Theme, App Icon).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `Accessibility` and notice the Accessibility is opened (as in Production).
- [ ] Tap on `About` and notice the updated About is opened (icon and copy at the top, Privacy Policy, Version, Share Feedback).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `DuckDuckGo Mac App` and notice the DuckDuckGo Mac screen is opened (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] Tap on `DuckDuckGo Windows App` and notice the DuckDuckGo Windows screen is opened (as in Production).
- [ ] Go back to the Settings screen. 
- [ ] All internal features as unchanged.

### UI changes
See https://app.asana.com/0/0/1204969193599390/f

### CODE REVIEW

**Comment by anikiki** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3226** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3226" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3226?utm_source=stack-comment).

**Review by malmstein — COMMENTED** [review]
A few nits and questions:
- Did you consider using the new Nav library instead?
- Are some pixels still missing?

File: app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    fun viewState(): StateFlow<ViewState> {
+        return viewState
+    }
+
+    fun commands(): Flow<Command> {
+        return command.receiveAsFlow()
+    }
+
+    fun onStartActivityCalled() {
```
[the remark this answers is not in the document]
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt:73]: Thanks, updated in the other PR.

**Comment by anikiki** [comment]
> A few nits and questions:
> 
>     * Did you consider using the new Nav library instead?
> 
>     * Are some pixels still missing?

Updated in the other PR to use the Nav library. Added all pixels.

File: app/src/main/res/layout/activity_private_search.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                android:id="@+id/privateSearchHeadingSearchSettings"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="@string/privateSearchHeadingSearchSettings" />
+
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/privateSearchAutocompleteToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
```
Comment (malmstein) [inline app/src/main/res/layout/activity_private_search.xml:90]: nit: 
```suggestion
                android:layout_marginTop=“@dimen/keyline_4"
```

**Review by malmstein — APPROVED** [review]
Great work, I left just a suggestion for a dimension. Works as expected.

**Pull request merged**